src/main/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionController.java

```diff
@@ -196,8 +198,8 @@ public ResponseEntity patchObjection(
         );
 
         try {
-            ServiceResult<String> result = objectionService.addAttachment(requestId, file);
-            return new ResponseEntity(result.getData(), HttpStatus.OK);
+            objectionService.addAttachment(requestId, objectionId, file, servletRequest.getRequestURI());
+            return new ResponseEntity(HttpStatus.NO_CONTENT);
```

Comment:
An HTTP 201 code rather than NO_CONTENT should be returned in this scenario, according to the spec.